.travis.yml

```diff
@@ -16,4 +16,5 @@ before_script:
   - composer self-update
   - composer install --no-interaction
 
-script: cd tests/suites/ && php all.php
+script: ./vendor/bin/phpunit --bootstrap ./vendor/autoload.php  ./tests/cases/
```

Comment:
Instead of relying on CLI arguments this should instead add a configuration file, preferably `phpunit.xml.dist` (or `phpunit.xml` just in case the ancient 4.8 release doesn't read the dist file yet)